tests/integration/test_ses.py

```diff
@@ -13,6 +17,10 @@
 
 class SESTest(unittest.TestCase):
 
+    @classmethod
+    def setUpClass(cls):
+        ses_starter.apply_patches()
```

Comment:
This should not be required - the patches should get applied automatically on startup (as part of the integration test startup), let's better remove it from here.

Reply:
Ah, dammit, could be a left-over. (Background: I first thought it would be enough to patch the email_responses package method sen_email, however, it was too complicated as the return value is already a XML response. I think I needed it back then).

I'll remove that :)